src/librustdoc/lib.rs

```diff
@@ -338,6 +338,11 @@ fn opts() -> Vec<RustcOptGroup> {
                        "enable-index-page",
                        "To enable generation of the index page")
         }),
+        unstable("persist-executable", |o| {
```

Comment:
Bikeshed alert: I feel like `--persist-doctests` is a better name, since the `executable` here feels a little ambiguous if there's a binary crate somewhere in the user's mind that could get confused with this.

Reply:
Haha, no idea how I missed that. Definitely agree with you there.